frontend/src/metabase/dashboard/actions/getNewCardUrl.ts

```diff
@@ -80,18 +102,20 @@ export function getParametersMappedToDashcard(
   dashcard: QuestionDashboardCard,
 ): ParameterWithTarget[] {
   const { parameter_mappings } = dashcard;
-  return (parameters || [])
-    .map(parameter => {
-      const mapping = _.findWhere(parameter_mappings || [], {
-        parameter_id: parameter.id,
-      });
+  return (parameters || []).flatMap(parameter => {
+    const mapping = _.findWhere(parameter_mappings || [], {
+      parameter_id: parameter.id,
+    });
```

Comment:
```suggestion
  return (parameters || []).map(parameter => {
    const mapping = _.findWhere(parameter_mappings || [], {
      parameter_id: parameter.id,
    });
    
    if (!mapping) {
      return null;
    }

    return [
      {
        ...parameter,
        target: mapping.target,
      },
    ];
  })
  .filter(isNotNull);
```

Reply:
I actually changed it from `map` + `filter` to `flatMap` because it's more convenient